mne/cuda.py

```diff
@@ -275,8 +320,10 @@ def _setup_cuda_fft_resample(n_jobs, W, new_len):
             try:
                 import cupy
 
+                W = _share_cuda_mem(W, n_jobs)
```

Comment:
```suggestion
                W = _share_cuda_mem(W, "cuda")
```

Reply:
yea I missed earlier, n_jobs=1 right above this. 

So this change makes them all hardcode "cuda".  do you still want `n_jobs` as a param here for future use?

Reply:
nm, hfft still passes n_jobs.  ok cool

Reply:
If it's `n_jobs="cuda"` everywhere then no we shouldn't need it! And if we already know we're cuda capable to be in that code path we can remove that part of the conditional, too. So maybe it just ends up being a numba conditional... simplify as it makes sense.